sessionctx/variable/tidb_vars.go

```diff
@@ -404,6 +404,9 @@ const (
 	// tidb_enable_window_function is used to control whether to enable the window function.
 	TiDBEnableWindowFunction = "tidb_enable_window_function"
 
+	// tidb_enable_pipelined_window_function is used to control whether to use pipelined window function, it only works when tidb_enable_window_function = true.
+	TiDBEnablePipelinedWindowFunction = "tidb_enable_pipelined_window_function"
```

Comment:
can we hide this variable by default since we are going to add enough tests to make it GA?

Reply:
sure, but for now this PR is not ready for reviewing, and I hide it by default before we want to merge it.

Reply:
resolved by add TiDBEnablePipelinedWindowFunction to FeatureSwitchVariables